src/org/typo3/chefci/helpers/JenkinsHelper.groovy

```diff
@@ -0,0 +1,24 @@
+package org.typo3.chefci.helpers
```

Comment:
looks good to me! we can use it until the namespace gets "too big" and add sub-namespaces later on...